### PR TITLE
design sprint announcement post

### PR DESCRIPTION
**Pull request by kidfribble**

### CODE REVIEW

File: source/posts/2015-10-05-announcing-design-sprint-guide.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+title: "Announcing the Design Sprints Guide"
+author: 'Steven Trevathan'
+twitter: 'strevat'
+social: true
+summary: 'Collaborative documentation for the road ahead'
+published: true
+tags: design, design process, user experience
+---
+
+We’re extremely excited to announce the release of our initial version of the Design Sprints Guide. This releases focuses on a special 4-day variant of the ever popular rapid-validation product design methodology.
```
Comment (atreni) [inline source/posts/2015-10-05-announcing-design-sprint-guide.md]: initial - feels like a strange word choice to me. maybe - "of our very first Design Sprints Guide"

releases - should be release

File: source/posts/2015-10-05-announcing-design-sprint-guide.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+In this guidebook we provide everything you can (and cannot) expect of the methodology, the processes and tools you can use for running a design sprint yourself, and a boat-load of resources for further reading.
+
+This content is free (so you don’t have to buy some immediately out-of-date book) and we’d love for those already running sprints to contribute wisdom from their experiences. For now, anyone who’d like to make edits, suggestions, or additions to this guide may send pull requests or open issues on the github repository. The README has some instructions for how we’ll manage these.
```
Comment (atreni) [inline source/posts/2015-10-05-announcing-design-sprint-guide.md]: remove - (so you don't have to buy some immediately out-of-date book) {just my opinion but it feels unnecessary}

File: source/posts/2015-10-05-announcing-design-sprint-guide.md (hunk 1 above)
Comment (kidfribble) [inline source/posts/2015-10-05-announcing-design-sprint-guide.md]: I'll probably write "initial release" or "first version". Point is that it is a living document, not one and done.

File: source/posts/2015-10-05-announcing-design-sprint-guide.md (hunk 2 above)
Comment (kidfribble) [inline source/posts/2015-10-05-announcing-design-sprint-guide.md]: agree - not necessary

**Pull request merged**